File: backend/components/metiers/administration_publique/utils/ai/helpers/ai_helper.test.js
// ai_helper.test.js
// Tests unitaires JS pour ai_helper
const { normalizeText } = require('./ai_helper');

describe('normalizeText', () => {
  it('normalise un texte avec accents et espaces', () => {
    expect(normalizeText('  Héllo   Wôrld  ')).toBe('Hello World');
  });
  it('gère le cas vide', () => {
    expect(normalizeText('')).toBe('');
  });
});

File: backend/components/metiers/administration_publique/utils/ai/helpers/ai_helper.test.py
# ai_helper.test.py
# Tests unitaires Python pour ai_helper
from .ai_helper import normalize_text

def test_normalize_text():
    assert normalize_text('  Héllo   Wôrld  ') == 'Hello World'

def test_normalize_text_empty():
    assert normalize_text('') == ''
